MTGSDKSwift/MTGAPIManager.swift

```diff
@@ -122,13 +61,16 @@ final class NetworkOperation {
                 networkError = NetworkError.requestError(error!)
                 return
             }
-            
+            guard data != nil else {
+                networkError = NetworkError.miscError("Network operation - No data returned")
+                return
+            }
+
             if let httpResponse = (response as? HTTPURLResponse) {
+                print("MTGSDK HTTPResponse - status code: \(httpResponse.statusCode)")
```

Comment:
Can we leave the `if Magic.enableLogging {` in place so the library is only verbose if we tell it to be?  I'm open to other ideas (for example a global `debugPrint` function that has that debug check built into the implementation).